Assets/_Project/Code/Managers/categoryDrawManager.cs

```diff
@@ -86,20 +71,27 @@ private void AwardBiddingWinners(TeamManager team, string categoryName)
         }
         else //wylosowano kategorie pytaniowa
         {
+
             //todo tutaj wywolac metode rozpoczynajaca etap pytania
```

Comment:
To jest dalej todo?

Reply:
Nie zauważyłem tego i wrzuciłem to w innym miejscu.

Reply:
Chyba jest to już do wyrzucenia.

Reply:
Zostało zaimplementowane, a niepotrzebny zakomentowany kod usunięty.